### PR TITLE
Fix race ethnicity

### PR DESCRIPTION
**Pull request by saulakravitz**
- add input race and ethnicity to birth and fetal death compositions
- add input race and ethnicity to examples
- Fix referenced codesystem in concept maps

### CODE REVIEW

File: input/fsh/bundlescompositions/CompositionJurisdictionFetalDeathReportNew.fsh
Comment (ptran265) [inline input/fsh/bundlescompositions/CompositionJurisdictionFetalDeathReportNew.fsh:1]: should this also have a slice added for FathersRaceEthnicity?

File: input/fsh/bundlescompositions/CompositionProviderFetalDeathReportNew.fsh
Comment (ptran265) [inline input/fsh/bundlescompositions/CompositionProviderFetalDeathReportNew.fsh:1]: also here - should this also have a slice for fathers race and ethnicity?

Comment (saulakravitz) [reply input/fsh/bundlescompositions/CompositionProviderFetalDeathReportNew.fsh:1]: No.  Race and Ethnicity is only done for the father for the live birth.

File: input/fsh/bundlescompositions/CompositionJurisdictionFetalDeathReportNew.fsh
Comment (saulakravitz) [reply input/fsh/bundlescompositions/CompositionJurisdictionFetalDeathReportNew.fsh:1]: No. Race and Ethnicity is only done for the father for the live birth.

**Review by ptran265 — APPROVED** [review]

**Pull request merged**